docker-compose.yaml

```diff
@@ -1,5 +1,16 @@
 version: "3.8"
 services:
+  flyway:
+    image: flyway/flyway:10
+    container_name: flyway-cont
```

Comment:
pls do not use the name of the library/etc in the container name - use the application name. what if you change from flyway to liquibase or something else?